src/sinks/papertrail.rs

```diff
@@ -111,7 +110,10 @@ impl SinkConfig for PapertrailConfig {
     }
 
     fn input(&self) -> Input {
+        let requirement = schema::Requirement::empty().optional_meaning("host", Kind::bytes());
```

Comment:
Similar query about a host optional meaning helper function, as from the other PR regarding timestamp. Again, might be overkill 🤷